include/NovelRT/Audio/AudioService.h

```diff
@@ -7,8 +7,8 @@
 #error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
 #endif
 
-//TODO: This won't exist after Kenny's rewrite, not too bothered about this class.
 namespace NovelRT::Audio {
+  // TODO: This won't exist after Kenny's rewrite, not too bothered about this class.
```

Comment:
Because **this** happened:

![image](https://user-images.githubusercontent.com/16415180/77784528-f485d300-705a-11ea-9c04-03efb7e864aa.png)